packages/blade/src/components/Typography/_decisions/decisions.md

```diff
@@ -130,7 +130,11 @@ Type of `children: React.ReactNode`
   Text will internally use `BaseText` to render. Natively we'll use html `<code></code>` tag for web and native `<Text></Text>` react-native to render the code text.
 
   ```jsx
-  <Code variant="large | caption" type="normal | subtle | subdued | muted | placeholder">
+  <Code size="large | medium" type="normal | subtle | subdued | muted | placeholder">
     SENTRY_AUTH_TOKEN:xyz
   </Code>
   ```
+
+  Using `size` here because that is what we use in `Text` as well and symantically `size="large"` makes a lot more sense.
+
+  In future, we want to make `size` as a consistent prop for sizing all components and use `variant` for values that can't be added to size.
```

Comment:
We had a discussion around this here - https://razorpay.slack.com/archives/G01B3LQ9H0W/p1663325670139239?thread_ts=1663318580.283669&cid=G01B3LQ9H0W

We would want to standardise the `size` for all typography components. Do let me know the thoughts.